packages/app-root/src/app/robots.js

```diff
@@ -0,0 +1,19 @@
+const appEnv = process.env.APP_ENV || "development";
+
+export default function robots() {
+  const isProduction = appEnv === "production";
```

Comment:
Does Next.js pass the host name to route handlers? If so, this could be changed to check if the host is `www.zooniverse.org`, which might be easier to follow.